manifests/image-references

```diff
@@ -6,6 +6,10 @@ spec:
     from:
       kind: DockerImage
       name: registry.ci.openshift.org/openshift:cluster-capi-operator
+  - name: cluster-capi-controllers
```

Comment:
I don't see cluster-capi-controllers within the OCP build data, I would expect it needs to be here before we can merge an image reference: https://github.com/openshift/ocp-build-data/tree/openshift-4.10/images 

While this may work for CI, merging without proper build data could break the nightly build process

Reply:
https://github.com/openshift/ocp-build-data/blob/openshift-4.10/images/ose-cluster-api.yml, there is a config for capi

Reply:
I removed new image references from this PR, need to figure out what is wrong.